graylog2-web-interface/src/components/navigation/SystemMenu.jsx

```diff
@@ -149,13 +149,13 @@ const SystemMenu = ({ location }) => {
       <IfPermitted permissions={['dashboards:create', 'inputs:create', 'streams:create']}>
```

Comment:
Thanks a lot for noticing and fixing these permissions @ryan-carroll-graylog!

This looks good to me. Just noticed on thing: I think these content pack permission checks might also need to be swapped out with just `contentpack:read`. Perhaps when these individual dashboards/inputs/streams permissions checks were added, these were the only entities supporting content packs. My personal feeling is that checking the content packs read permission here is probably enough, since it's probably not maintainable (or pluggable) to check individual entity permissions here. Those permissions checks should probably happen on the Content Packs pages. I am not sure if that is being done right now, but that might be worth checking as well, just to see if there is any issue with permissions checks not being done (if so, we might want to capture that in another issue).

Reply:
Ok good call @danotorrey, I hadn't realize that was an option. I just double checked the permissions on the Contents pack and a user with only `contentpack:read` can view the page but they can't install a pack with entities they don't have permissions for or create a Content pack.

I also pared down the Lookup Table permissions to just UPGRADING.md. They can view LUTs, Caches, and Data Adapters but can't make changes without the other permissions.

I also added documentation to UPGRADING.md. Let me know if you think that's specific enough and makes sense.

Reply:
LGTM! Thanks @ryan-carroll-graylog!